spec/features/proposal_spec.rb

```diff
@@ -127,6 +128,7 @@
     end
 
     scenario 'confirms a proposal', feature: true, js: true do
+      registration_period # opens conference for registration
```

Comment:
``
let!(:registration_period) { create(:registration_period, conference: conference, start_date: Date.current) }
``
(note the exclamation mark) does not work?